### ISSUE TITLE
`the build` incorrectly parsing `}`, `/*!`, etc

### ISSUE THREAD

**Issue by delasy** [thelang-io/cli#5]

**Comment by delasy** [comment]
CLI builder is right now limited and can't do a 'real' parsing, so it pretends that it can. If you encounter bugs like:

```the
test := "/*!"
```

Being broken, just do like this:

```the
test := "/" + "*!"
```

Or if your code has curly braces, for example:

```the
test := "main {"
```

This will also break builder, possible solution:

```the
test := "main {" // }
```

This problem will go away once we have modules system that will roll in v0.14

**Comment by delasy** [comment]
Potentially could be solved by using git submodules

**Comment by delasy** [comment]
fixed in 0.14
